services/frontend/src/scenes/Profile/ProfileLayout.js

```diff
@@ -1,77 +1,30 @@
-                sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
-                rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
-                eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
-                cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
-                i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
-
-                Occaecat reprehenderit fugiat qui ullamco ad commodo Lorem velit nisi aliquip sit esse officia con
-                sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
-                rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
-                eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
-                cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
-                i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
-
-                <br /><br />
-
-                Occaecat reprehenderit fugiat qui ullamco ad commodo Lorem velit nisi aliquip sit esse officia con
-                sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
-                rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
-                eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
-                cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
-                i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
-
-                Occaecat reprehenderit fugiat qui ullamco ad commodo Lorem velit nisi aliquip sit esse officia con
-                sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
-                rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
-                eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
-                cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
-                i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
-            </Paper>
+    <div className="profile-content">
+        <section id="profile-personal-details">
+            <ProfileDetails />
         </section>
-        <section id="skills">
+
+        <section id="profile-skills">
             <h2>Skills</h2>
-            <Paper className="profile-content-card">
+            <Paper className="profile-card">
```

Comment:
The styling we are doing is relatively simple but also doesn't make use of a lot of the default styling/features of `Card`, figured it makes sense to go with the lightweight `Paper` instead of `Card` because of that.

Reply:
Makes sense. I think I'll convert some of my stuff in the Projects list over to `Paper` for consistency's sake.